src/config.ts

```diff
@@ -93,7 +93,11 @@ export interface QueryConfig {
   maxCardinalityForShape?: number;
   typeMatchesSchemaType?: boolean;
 
-  // Effectiveness Preference
+  // STYLIZE
+  smallBandSizeForHighCardinalityOrFacet?: {maxCardinality: number, bandSize: number};
+  nominalColorScaleForHighCardinality?: {maxCardinality: number};
```

Comment:
add `palette: string`, which is default to `categorical20`.  
(But user can choose other config as well.)